wp-parsely.php

```diff
@@ -154,3 +154,54 @@ function parsely_integrations(): Integrations {
 
 	return $parsely_integrations;
 }
+
+/**
+ * Helper function to determine if this plugin is activated "network-wide" for a Multisite instance.
+ *
+ * @return boolean
+ */
+function is_network_active(): bool {
+	return is_multisite() && is_plugin_active_for_network( _plugin_basename() );
+}
```

Comment:
This functions do make sense, but maybe let's move them somewhere else than `wp-parsely.php`